components/camel-bindy/src/main/java/org/apache/camel/dataformat/bindy/annotation/DataField.java

```diff
@@ -146,4 +146,9 @@
      */
      String rounding() default "CEILING";
 
+     /**
+      * Method name to call to apply such customization
+      * on DataField
```

Comment:
Add explanation that if you provide a FQN class name then its invoking that method (and it must be a static method) and if you do not provide a FQN its invoking the method on the value itself, eg toUpperCase on a String value etc.

Reply:
tried to address